lib/localization/locales/tr.json

```diff
@@ -0,0 +1,63 @@
+{
+  "loading": "Yükleniyor...",
+  "why_am_i_seeing": "Bunu neden görüyorum?",
+  "protected_by": "Koruma sağlayan",
+  "made_with": "🇨🇦'da ❤️ ile yapıldı",
+  "mascot_design": "Maskot tasarımı:",
+  "ai_companies_explanation": "Bunu görüyorsunuz çünkü bu web sitesinin yöneticisi, AI şirketlerinin web sitelerini agresif şekilde kazımasına karşı sunucuyu korumak için Anubis'i kurdu. Bu durum sitelerin erişilemez olmasına ve kesintilere neden olabiliyor.",
+  "anubis_compromise": "Anubis bir uzlaşmadır. Anubis, spam e-postaları azaltmak için önerilen bir iş kanıtı sistemi olan Hashcash benzeri bir sistemi kullanır. Bireysel kullanımda bu ek yük göz ardı edilebilir olsa da, büyük ölçekli kazıyıcılarda birikerek kazımayı oldukça maliyetli hale getirir.",
+  "hack_purpose": "Bu geçici bir çözümdür. Esas amacı, başsız tarayıcıları parmak iziyle tanımlamak için daha fazla zaman kazandırmak, ve bu sayede meşru kullanıcıların bu zorluk sayfasını görmesini önlemektir.",
+  "jshelter_note": "Lütfen dikkat: Anubis, JShelter gibi eklentilerin devre dışı bıraktığı modern JavaScript özelliklerini gerektirir. Lütfen bu alan adı için JShelter veya benzeri eklentileri devre dışı bırakın.",
+  "version_info": "Bu web sitesi şu Anubis sürümünü çalıştırıyor:",
+  "try_again": "Tekrar dene",
+  "go_home": "Ana sayfaya dön",
+  "contact_webmaster": "ya da engellenmemeniz gerektiğini düşünüyorsanız, lütfen şu adrese e-posta gönderin:",
+  "connection_security": "Bağlantınızın güvenliğini sağlarken lütfen bekleyin.",
+  "javascript_required": "Ne yazık ki bu aşamayı geçebilmek için JavaScript’i etkinleştirmeniz gerekiyor. Bunun nedeni, AI şirketlerinin web barındırma konusundaki sosyal sözleşmeyi değiştirmiş olmasıdır. JavaScript’siz bir çözüm geliştirilmektedir.",
+  "benchmark_requires_js": "Kıyaslama aracının çalıştırılması için JavaScript’in etkin olması gereklidir.",
+  "difficulty": "Zorluk:",
+  "algorithm": "Algoritma:",
+  "compare": "Karşılaştır:",
+  "time": "Süre",
+  "iters": "Tekrar",
+  "time_a": "Süre A",
+  "iters_a": "Tekrar A",
+  "time_b": "Süre B",
+  "iters_b": "Tekrar B",
+  "static_check_endpoint": "Bu sadece ters proxy'nizin kullanması için bir kontrol adresidir.",
```

Comment:
If we are aiming completely Turkish works maybe "vekil" or something like that works.